<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="sysadm-Latitude-3480" name="paymentsAPI.API_EW_Pay_V3" tests="2" failures="1" timestamp="05 Mar 2020 05:12:33 GMT" time="12.074" errors="0">
  <testcase name="Expressway_Pay" time="0.166" classname="paymentsAPI.API_EW_Pay_V3">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:48)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at paymentsAPI.API_PaymentCommon1.validation(API_PaymentCommon1.java:1527)
at paymentsAPI.API_EW_Pay_V3.Expressway_Pay(API_EW_Pay_V3.java:19)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- Expressway_Pay -->
  <testcase name="Expressway_Pay" time="11.908" classname="paymentsAPI.API_EW_Pay_V3">
    <skipped/>
  </testcase> <!-- Expressway_Pay -->
</testsuite> <!-- paymentsAPI.API_EW_Pay_V3 -->
